lib/optimizely/tests.js

```diff
@@ -229,15 +313,20 @@ describe('lib/optimizely', function() {
       sinon.stub(eventDispatcher, 'dispatchEvent').returns(bluebird.resolve());
```

Comment:
Let's make it not return a promise so that we can ensure that the default event dispatcher does execute the callback.